dashboard/app/controllers/plc/tasks_controller.rb

```diff
@@ -8,7 +8,7 @@ def show
 
```

Comment:
The entry point for task creation is still from the learning module page. You might as well start with the learning_module_id preselected